common/arg.cpp

```diff
@@ -2447,6 +2447,13 @@ common_params_context common_params_parser_init(common_params & params, llama_ex
             params.vocoder.use_guide_tokens = true;
         }
     ).set_examples({LLAMA_EXAMPLE_TTS, LLAMA_EXAMPLE_SERVER}));
+    add_opt(common_arg(
+        {"--tts-speaker-file"}, "FNAME",
+        "speaker file path for audio generation",
+        [](common_params & params, const std::string & value) {
+            params.vocoder.speaker_file = value;
+        }
+    ).set_examples({LLAMA_EXAMPLE_TTS, LLAMA_EXAMPLE_SERVER}));
```

Comment:
```suggestion
    ).set_examples({LLAMA_EXAMPLE_TTS}));
```

I don't think llama-server supports this for now

Reply:
Thank you for your suggestion. I have removed `LLAMA_EXAMPLE_SERVER`.